src/components/PagingPanel/index.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+import classNames from 'classnames';
+import uuid from 'random-uuid-v4';
+
+export default class PagingPanel extends Component {
+    constructor(props) {
+        super(props);
+
+        console.log('here');
+        this.gap = props.gap || 5;
+        this.startPage = props.startPage || 1;
+        this.state = {
+            active: props.active || 1,
+            visibleStartPage: this.getVisibleStartPage()
+        };
+        console.log(this.state);
+    }
+
+    getVisibleStartPage() {
+        const {gap, startPage} = this;
+        const active = this.props.active;
+
+        let visibleStartPage = startPage;
+        while(visibleStartPage + gap < active) {
+            visibleStartPage += gap;
+        }
+
+        return visibleStartPage;
+    }
+
+    render() {
+        return (
+            <div className="buttons has-addons is-centered">
+                {this.getArrowLeft()}
+                {this.getPagesGroup()}
+                {this.getArrowRight()}
+            </div>
+        );
+    }
+
+    getArrowLeft = () => this.props.count > 5 && (
+        <button
+            className="button" 
+            onClick={this.handlePreviousPagesClick}>
+            <i className="material-icon" aria-hidden="true">chevron_left</i>
+        </button>
+    )
+
+    handlePreviousPagesClick = () => {
+        let newVisibleStartPage = this.state.visibleStartPage - this.gap;
+        newVisibleStartPage = newVisibleStartPage > this.startPage ? 
```

Comment:
When you have operator in the begging of string it more readable.
`
newVisibleStartPage > this.startPage
    ? newVisibleStartPage
    : this.startPage;
`